WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -210,12 +211,37 @@ - (void)updateSearchVisibility
 
 - (void)maybeShowNUX
```

Comment:
Is this method name correct? NUX is a concept or a specific VC?

Reply:
NUX is a concept of the "New User Experience", the more up to date naming is now onboarding if I remember correctly. 

I'm not sure I want to rename this method in this PR since I feel it's a bit out of scope of what I'm doing. Thoughts?

Reply:
I misunderstood then. Let's keep it.